R/pkg/R/functions.R

```diff
@@ -202,8 +202,9 @@ NULL
 #'          \itemize{
 #'          \item \code{from_json}: a structType object to use as the schema to use
 #'              when parsing the JSON string. Since Spark 2.3, the DDL-formatted string is
-#'              also supported for the schema.
-#'          \item \code{from_csv}: a DDL-formatted string
+#'              also supported for the schema. Since Spark 3.0, \code{schema_of_json} or
+#'              a string literal can also be accepted.
```

Comment:
What `a string literal` means here? Is it different to DDL-formated string?

Reply:
that's true...? they are the same?

Reply:
Basically yea same. The only difference is it allows takes a string literal (not only string). Let me try to clarify it.